nls/platform.properties

```diff
@@ -608,6 +616,8 @@ tooltip.creation.app.settings.existingRule=If available in the application names
 tooltip.creation.app.settings.localClusters=Deploy application resources on local cluster only
 tooltip.creation.app.settings.onlineClusters=Deploy your application resources on all online clusters, including your local cluster.
 tooltip.creation.app.settings.onlineClustersOnly=Deploy your application resources on all online clusters.
+tooltip.creation.app.reconcileRate = The frequeny of resource reconciliation that will be used as a global repository setting. The default setting is medium that checks for changes to apply every 3 minutes and re-applies all resources ever 15 minutes even if there is no change. Select low to reconcile every hour and high to reconcile every 2 minutes. By selecting off, the deployed resources are not automatically reconciled.
```

Comment:
frequency misspelled here

Reply:
change: that will be used
to 
that is used

Reply:
The `medium` default setting checks for changes to apply every three minutes and re-applies all resources every 15 minutes, even without a change. Select `low` to reconcile every hour.  Select `high` to reconcile every two minutes. If you select `off`, the deployed resources are not automatically reconciled.

Many changes needed -- a bit confusing on the first line (subject/predicate changes), other small fixes.